guide/private-key-management/single-user-schemes.md

```diff
@@ -53,9 +53,9 @@ Now let's dive in and look at the various schemes that might be suitable for a p
 
 By using an automatic cloud backup for the private key the user is NOT required to manually backup anything when setting up a new [bitcoin wallet]({{ '/guide/glossary/#bitcoin-wallet' | relative_url }}). This is sometimes also called *seedless*, as the user is never exposed to the traditional *seed*, or [recovery phrase]({{ '/guide/glossary/#recovery-phrase' | relative_url }}) that is used to initiate a wallet.
 
-This scheme can be used for those new to bitcoin, or novices who would find manually backing up a recovery phrase during onboarding confusing, possibly leading to unsafe backups. It could also be a starting point for a progressive security model that lets customers upgrade when they start accumulating more funds.
+Automatic cloud backups can be useful for a variety of user experience levels. New Bitcoin users will likely find this method more frictionless. For new users, manual backups may actually be less secure as the process is likely foreign to them. This can lead to writing down or storing the backup in an unsafe manner. More experienced users can also make use of this scheme.  Wallets that store private keys locally, also known as hot or spending wallets, likely don't need the security manual backups offer. This scheme may also bridge the gap between new and experienced users. A security model may start new users with an automatic cloud backup. Then over time shift users to more secure scheme as they accumulate more bitcoin.
```

Comment:
```suggestion
This scheme can be a good fit for wallets meant for storing smaller amounts of bitcoin. This includes wallets for novices who would find manually backing up a recovery phrase during onboarding confusing, possibly leading to unsafe backups. It could also be a starting point for a progressive security model that lets customers upgrade when they start accumulating more funds. Wallets using automatic cloud backups also work well for what experienced users might call *hot* or *spending* wallets. 
```

Reply:
Saying the same thing with fewer words.

Reply:
Cheers for that :)